docs/rules/no-zero-fractions.md

```diff
@@ -7,7 +7,7 @@
 <!-- end auto-generated rule header -->
 <!-- Do not manually modify this header. Run: `npm run fix:eslint-docs` -->
 
-There is no difference in JavaScript between, for example, `1`, `1.0` and `1.`, so prefer the former for consistency.
+There is no difference in JavaScript between, for example, `1`, `1.0` and `1.`, so prefer the former for consistency and brevity.
```

Comment:
What's a word for "completely useless and redundant"? 😅

Reply:
`1..toString()`is not.useless.

Reply:
The rule doesn't affect that, so why are you mentioning it?

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/f14433068bbf92d56b3aaa457c5a1aa0ec19e7ab/test/snapshots/no-zero-fractions.mjs.md?plain=1#L2630